shuup/importer/transforms.py

```diff
@@ -125,7 +127,16 @@ def py2_read_file(data, filename):
 def py3_read_file(data, filename):
     got_data = set()
     data = []
-    with open(filename, encoding="utf-8") as f:
+
+    bytes = min(32, os.path.getsize(filename))
+    raw = open(filename, 'rb').read(bytes)
+
+    if raw.startswith(codecs.BOM_UTF8):
+        encoding = 'utf-8-sig'
+    else:
+        encoding = "utf-8"
+
+    with open(filename, encoding=encoding) as f:
```

Comment:
How about the py2 reader? Why do we have different py2 and py3 versions anyway?

However.. at least there isn't enough tests, since tests didn't find out that py2 version doesn't strip the BOM.